website/servers/manage.php

```diff
@@ -0,0 +1,349 @@
+<?php
+
+include_once __DIR__ . '/../../config/config.inc.php';
+include_once __DIR__ . '/../../config/servers.inc.php';
+include_once '../../lib/ntbb-session.lib.php';
+include '../style/wrapper.inc.php';
+include_once __DIR__ . '/../../lib/ntbb-database.lib.php';
+include_once __DIR__ . '/servers.lib.php';
```

Comment:
```suggestion
include_once './servers.lib.php';
```

All of these are already dependencies of `servers.lib.php`.